jupyterlite_xeus_python/env_build_addon.py

```diff
@@ -52,6 +52,10 @@ class XeusPythonEnv(FederatedExtensionAddon):
         description="The path or URL to the empack config file",
     )
 
+    pin_packages = PackagesList([]).tag(
```

Comment:
Why was `packages` not enough for Voici?

Also regarding the name, how about `extra_packages` so it's coherent with other config options?

Reply:
> Why was packages not enough for Voici?

Because it can (and it will) be overwritten by users. 

This property I added should not be part of the config/public API.